src/components/Interpretations/InterpretationModal/InterpretationThread.js

```diff
@@ -52,13 +62,13 @@ const InterpretationThread = ({
                                 currentUser={currentUser}
                                 interpretationId={interpretation.id}
                                 onThreadUpdated={onThreadUpdated}
-                                interpretationWriteAccess={
-                                    interpretation.access.write
+                                hasInterpretationReplyAccess={
+                                    interpretationAccess.reply
```

Comment:
Should we use `reply` instead of `comment` everywhere?

Reply:
ah, you mean since the component is `Comment`

Reply:
Done